data/data/bootstrap/baremetal/README.md

```diff
@@ -22,19 +22,37 @@ Relevant files:
 * **systemd/units/keepalived.service** - systemd unit file for `keepalived`.
   This runs `keepalived.sh` to generate the proper configuration from the
   template and then runs podman to launch `keepalived`.
+* **files/usr/local/bin/fletcher8** - Script that uses the
+  [fletcher8](https://en.wikipedia.org/wiki/Fletcher%27s_checksum) algorithm to
+  generate a hash from an input string. This is used to generate VRRP ids for
+  use with Keepalived.  This is used by `keepalived.sh`
+* **files/usr/local/bin/get_vip_subnet_cidr** - Script to determine the network
+  CIDR for a given VIP.  This is used by `keepalived.sh`
```

Comment:
which is in turn used to determine which local interface is on the VIP's network

Reply:
added, thanks!